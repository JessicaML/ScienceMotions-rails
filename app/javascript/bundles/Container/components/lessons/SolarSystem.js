import React, { Fragment } from "react";
import Text from "./text/Text";

class SolarSystem extends React.Component {
  constructor(props) {
    super(props);
    };
  


  render() {
    return (
      <Fragment>
        <section id="visual">
           
            <div  className="clearfix">
            <ul className="solarsystem">
                <li className="sun" >
                    <a href="#sun">
                        <span>Sun</span>
                    </a>
                </li>
                <li className="mercury">
                    <a href="#mercury">
                        <span>Mercury</span>
                    </a>
                </li>
                <li className="venus">
                    <a href="#venus">
                        <span>Venus</span>
                    </a>
                </li>
                <li className="earth">
                    <a href="#earth">
                        <span>Earth
                            <span className="moon"> &amp; Moon</span>
                        </span>
                    </a>
                </li>
                <li className="mars">
                    <a href="#mars">
                        <span>Mars</span>
                    </a>
                </li>
                <li className="asteroids_meteorids">
                    <span>Asteroids &amp; Meteorids</span>
                </li>
                <li className="jupiter">
                    <a href="#jupiter">
                        <span>Jupiter</span>
                    </a>
                </li>
                <li className="saturn">
                    <a href="#saturn">
                        <span>Saturn &amp;
                            <span className="ring">Ring</span>
                        </span>
                    </a>
                </li>
                <li className="uranus">
                    <a href="#uranus">
                        <span>Uranus</span>
                    </a>
                </li>
                <li className="neptune">
                    <a href="#neptune">
                        <span>Neptune</span>
                    </a>
                </li>
                <li className="pluto">
                    <a href="#pluto">
                        <span>Pluto</span>
                    </a>
                </li>
            </ul>
        </div>
            
        </section>
        <Text props={this.props.description} />
      </Fragment>
    );
  }
}
export default SolarSystem;
