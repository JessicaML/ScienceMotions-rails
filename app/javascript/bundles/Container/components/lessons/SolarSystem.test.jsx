import React from 'react';
import renderer from 'react-test-renderer';
import SolarSystem from './SolarSystem';

describe('SolarSystem', () => {
  it('renders correctly when there are no items', () => {
    const rendered = renderer.create(<SolarSystem />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
})