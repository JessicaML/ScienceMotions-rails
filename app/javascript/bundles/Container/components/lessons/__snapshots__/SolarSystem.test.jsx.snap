// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SolarSystem renders correctly when there are no items 1`] = `
Array [
  <section
    className="lesson-container solarsystem"
    id="visual"
  >
    <div
      className="clearfix"
    >
      <ul
        className="solarsystem"
      >
        <li
          className="sun"
        >
          <a
            href="#sun"
          >
            <span>
              Sun
            </span>
          </a>
        </li>
        <li
          className="mercury"
        >
          <a
            href="#mercury"
          >
            <span>
              Mercury
            </span>
          </a>
        </li>
        <li
          className="venus"
        >
          <a
            href="#venus"
          >
            <span>
              Venus
            </span>
          </a>
        </li>
        <li
          className="earth"
        >
          <a
            href="#earth"
          >
            <span>
              Earth
              <span
                className="moon"
              >
                 & Moon
              </span>
            </span>
          </a>
        </li>
        <li
          className="mars"
        >
          <a
            href="#mars"
          >
            <span>
              Mars
            </span>
          </a>
        </li>
        <li
          className="asteroids_meteorids"
        >
          <span>
            Asteroids & Meteorids
          </span>
        </li>
        <li
          className="jupiter"
        >
          <a
            href="#jupiter"
          >
            <span>
              Jupiter
            </span>
          </a>
        </li>
        <li
          className="saturn"
        >
          <a
            href="#saturn"
          >
            <span>
              Saturn &
              <span
                className="ring"
              >
                Ring
              </span>
            </span>
          </a>
        </li>
        <li
          className="uranus"
        >
          <a
            href="#uranus"
          >
            <span>
              Uranus
            </span>
          </a>
        </li>
        <li
          className="neptune"
        >
          <a
            href="#neptune"
          >
            <span>
              Neptune
            </span>
          </a>
        </li>
        <li
          className="pluto"
        >
          <a
            href="#pluto"
          >
            <span>
              Pluto
            </span>
          </a>
        </li>
      </ul>
    </div>
  </section>,
  <section
    id="text"
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": undefined,
        }
      }
    />
  </section>,
]
`;
